politeiawww/cmd/cmswww_payment/cmswww_payment.go

```diff
@@ -0,0 +1,322 @@
+
+package main
+
+import (
+	"bytes"
+	_ "encoding/gob"
+	"encoding/hex"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"net/http"
+	"net/url"
+	"os"
+	"path/filepath"
+
+	"github.com/decred/politeia/mdstream"
+	pd "github.com/decred/politeia/politeiad/api/v1"
+	"github.com/decred/politeia/politeiad/api/v1/identity"
+	"github.com/decred/politeia/politeiad/cache"
+	cachedb "github.com/decred/politeia/politeiad/cache/cockroachdb"
+	cms "github.com/decred/politeia/politeiawww/api/cms/v1"
+	www "github.com/decred/politeia/politeiawww/api/www/v1"
+	cmsdb "github.com/decred/politeia/politeiawww/cmsdatabase/cockroachdb"
+	"github.com/decred/politeia/politeiawww/sharedconfig"
+	"github.com/decred/politeia/util"
+)
+
+const (
+	defaultDBHost     = "localhost:26257"
+	defaultDBRootCert = "~/.cockroachdb/certs/clients/politeiawww/ca.crt"
+	defaultDBCert     = "~/.cockroachdb/certs/clients/politeiawww/client.politeiawww.crt"
+	defaultDBKey      = "~/.cockroachdb/certs/clients/politeiawww/client.politeiawww.key"
+
+	defaultIdentityFilename = "identity.json"
+
+	defaultRPCUser = "user"
+	defaultRPCPass = "pass"
+	defaultRPCHost = "127.0.0.1"
+)
+
+var (
+	defaultHomeDir      = sharedconfig.DefaultHomeDir
+	defaultDataDir      = sharedconfig.DefaultDataDir
+	defaultRPCCertFile  = filepath.Join(sharedconfig.DefaultHomeDir, "rpc.cert")
+	defaultIdentityFile = filepath.Join(sharedconfig.DefaultHomeDir, defaultIdentityFilename)
+
+	// Application options
+	testnet    = flag.Bool("testnet", false, "")
+	dataDir    = flag.String("datadir", defaultDataDir, "")
+	dbHost     = flag.String("dbhost", defaultDBHost, "")
+	dbCert     = flag.String("dbcert", defaultDBCert, "")
+	dbRootCert = flag.String("dbrootcert", defaultDBRootCert, "")
+	dbKey      = flag.String("dbkey", defaultDBKey, "")
+
+	rpcCert = flag.String("rpcert", defaultRPCCertFile, "")
+	rpcHost = flag.String("rpchost", defaultRPCHost, "")
+	rpcUser = flag.String("rpcuser", defaultRPCUser, "")
+	rpcPass = flag.String("rpcpass", defaultRPCPass, "")
+	rpcPort = flag.String("rpcport", pd.DefaultMainnetPort, "")
+
+	dIdentityFile = flag.String("identityfile", defaultIdentityFile, "")
+
+	network string // Mainnet or testnet3
+)
+
+const usageMsg = `cmswww_payment usage:
+
+  Application options
+    -testnet
+          Use testnet database
+    -datadir string
+          politeiawww data directory
+		      (default osDataDir/politeiawww/data)
+    -dbhost string
+          CockroachDB ip:port 
+			  (default localhost:26257)
+    -dbrootcert string
+          File containing the CockroachDB SSL root cert
+          (default ~/.cockroachdb/certs/clients/politeiawww/ca.crt)
+    -dbcert string
+          File containing the CockroachDB SSL client cert
+          (default ~/.cockroachdb/certs/clients/politeiawww/client.politeiawww.crt)
+    -dbkey string
+          File containing the CockroachDB SSL client cert key
+		  (default ~/.cockroachdb/certs/clients/politeiawww/client.politeiawww.key)
+	-rpccert string
```

Comment:
The spacing in this string literal is showing up funny because its mixing tabs and spaces, so when its being rendered to the console it shows up as:
```
cmswww_payment usage:

  Application options
    -testnet
          Use testnet database
    -datadir string
          politeiawww data directory
                      (default osDataDir/politeiawww/data)
    -dbhost string
          CockroachDB ip:port
                          (default localhost:26257)
    -dbrootcert string
          File containing the CockroachDB SSL root cert
          (default ~/.cockroachdb/certs/clients/politeiawww/ca.crt)
    -dbcert string
          File containing the CockroachDB SSL client cert
          (default ~/.cockroachdb/certs/clients/politeiawww/client.politeiawww.crt)
    -dbkey string
          File containing the CockroachDB SSL client cert key
                  (default ~/.cockroachdb/certs/clients/politeiawww/client.politeiawww.key)
        -rpccert string
                  File containing the RPC cert to communicate with politeiad
                  (default ~/.politeiawww/rpc.cert)
        -rpcuser string
                  RPC user name for privileged commands
                  (default user)
        -rpcpass string
                  RPC password for privileged commands
                  (default pass)
        -rpchost string
                  RPC Host for politeiad
                  (default 127.0.0.1)
        -rpcport string
                  RPC Port for politeiad
                  (default 49374 / 59374)
        -identityfile
                  Path to file containing the politeiad identity
                  (default ~/.politeiawww/identity.json)
```

This is also how it shows up in the README and how github renders it.  If you replace the tabs with two spaces it will fix it.